src/lexer.coffee

```diff
@@ -116,10 +117,18 @@ exports.Lexer = class Lexer
     if id is 'from' and @tag() is 'YIELD'
       @token 'FROM', id
       return id.length
-    if id is 'as' and @seenImport and (@tag() is 'IDENTIFIER' or @value() is '*')
-      @tokens[@tokens.length - 1][0] = 'IMPORT_ALL' if @value() is '*'
-      @token 'AS', id
-      return id.length
+    if id is 'as' and @seenImport
+      # loop hack
```

Comment:
It's not ideal, I basically wan't to reach the `@token 'AS', id` and return only for a select few token types. So basically if none of the conditionals run, I want to skip that part and not return. Since CS doesn't have `goto`, I used this as a hack.

It's possible to do it without a loop, but it would require more code.